src/Compilers/CSharp/Portable/Symbols/ConstantValueUtils.cs

```diff
@@ -77,19 +77,34 @@ internal static string UnescapeInterpolatedStringLiteral(string s)
         {
             var builder = PooledStringBuilder.GetInstance();
             var stringBuilder = builder.Builder;
-            int formatLength = s.Length;
-            for (int i = 0; i < formatLength; i++)
+            for (int i = 0, formatLength = s.Length; i < formatLength; i++)
```

Comment:
nit: I'd prefer reverting this change (ie. keeping the declaration for `int formatLength` above)

Reply:
i can move it out, though this definitely seems more idiomatic as the normal way to iterate indices.  this way the 'formatLength' value isn't in a higher scope than necessary.